automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -201,8 +201,7 @@ class WorkspaceApiSpec
         }(owner.makeAuthToken(billingScopes))
 
         exception.fields("statusCode").convertTo[Int] should equal(400)
-        exception.fields("message").convertTo[String] should startWith("Workspace creation failed. Error trying to create bucket ")
-        exception.fields("message").convertTo[String] should endWith regex(s" in Google project (.+) in region `${invalidRegion}`.".r)
+        exception.fields("message").convertTo[String] should startWith("Workspace bucket location must be a single region of format")
```

Comment:
Note that we already have test coverage [here](https://github.com/broadinstitute/rawls/blob/30555ad9af1b48c68cd6f32c3130a4e3dbd24d11/core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala#L992) for a create workspace request with an invalid region and [here](https://github.com/broadinstitute/rawls/blob/30555ad9af1b48c68cd6f32c3130a4e3dbd24d11/core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/WorkspaceApiServiceSpec.scala#L1435) for clone workspace request.